src/blocks/author-list/edit.js

```
@@ -3,313 +3,259 @@
+export default function Edit({
+	attributes,
+	setAttributes,
+	clientId,
+	insertBlocks,
+}) {
+	const {
+		authorIds,
+		authorRole,
+		maxAuthors,
+		displayStyle,
+		listStyle,
```

ESLint

[error] 'listStyle' is assigned a value but never used.

(no-unused-vars)